domain/domain.go

```diff
@@ -99,8 +99,8 @@ func (do *Domain) loadInfoSchema(handle *infoschema.Handle, usedSchemaVersion in
 		log.Errorf("[ddl] failed to load schema diff err %v", err)
 	}
 	if ok {
-		log.Infof("[ddl] diff load InfoSchema from version %d to %d, in %v",
-			usedSchemaVersion, latestSchemaVersion, time.Since(startTime))
+		log.Infof("[ddl] diff load InfoSchema from version %d to %d in %v, tbls %v",
```

Comment:
s/tbls/tableIDs:/

Reply:
It's better not to use abbreviation here. 
If we're reading this function, this is OK because we have the context information, but when it used in log, we have no idea what's the abbreviation for.

Reply:
And when OPS see this log, he might thought: "what the fuck?"